docs/documentation/components/table-expando-row.html

```diff
@@ -181,7 +181,7 @@ <h1>Lorem ipsum details</h1>
                     <td class="number">12</td>
                     <td>New</td>
                     <td>
-                      <button class="expando-button" data-close-label="Sluit details">
+                      <button class="expando-button" data-close-label="Sluit details" data-open-label="Open details">
```

Comment:
These changes are not necessary. The button already has the label for its current state (closed by default, or based on `aria-expanded` if it is present in the markup). Only the "other" label needs to be added as a data attribute. There should be some explanation about this in the docs already, but if not it would make sense to add it.